docs/usage/model_config.md

```diff
@@ -584,6 +585,45 @@ print(type(model2.pet))
 #> <class '__main__.Pet'>
 ```
 
+
+## Coerce Numbers to Strings
+
+Pydantic no longer allows number types (`int`, `float`, `Decimal`) to be coerced as type `str` by default.
+
+Set [`coerce_numbers_to_str=True`](../api/config.md#pydantic.config.ConfigDict.coerce_numbers_to_str) to enable coercing of numbers to strings.
+
+```py
+from pydantic import BaseModel, ConfigDict, ValidationError
```

Comment:
To support the below `Decimal` example...

```suggestion
from decimal import Decimal

from pydantic import BaseModel, ConfigDict, ValidationError
```

Reply:
Exactly:)